src/wasm/wasm-ir-builder.cpp

```diff
@@ -1783,6 +1791,18 @@ Result<> IRBuilder::makeStringSliceIter() {
   return Ok{};
 }
 
+Result<> IRBuilder::makeContNew(HeapType ct) {
+  if (!ct.isContinuation()) {
+    return Err{"expected continuation type"};
+  }
+  ContNew curr(wasm.allocator);
```

Comment:
Can you add a default constructor to `ContNew` so we can do this instead?
```suggestion
  ContNew curr;
```

Reply:
Done